CMakeLists.txt

```diff
@@ -194,7 +194,12 @@ KOKKOS_SETUP_BUILD_ENVIRONMENT()
 OPTION(BUILD_SHARED_LIBS "Build shared libraries" OFF)
 
 SET(KOKKOS_EXT_LIBRARIES Kokkos::kokkos Kokkos::kokkoscore Kokkos::kokkoscontainers Kokkos::kokkosalgorithms)
-SET(KOKKOS_INT_LIBRARIES kokkos kokkoscore kokkoscontainers kokkosalgorithms)
+SET(KOKKOS_SUB_LIBRARIES kokkos kokkoscore kokkoscontainers kokkosalgorithms)
```

Comment:
What is the rational for introducing `KOKKOS_SUB_LIBRARIES`?

Reply:
Later on the target `kokkos` is linked to `${KOKKOS_SUB_LIBRARIES}`, as a way to conditionally exclude the `kokkossimd` target from being linked to `kokkos`.